ui/components/component-library/text-field-base/text-field-base.js

```diff
@@ -107,7 +108,7 @@ export const TextFieldBase = ({
       backgroundColor={COLORS.BACKGROUND_DEFAULT}
       alignItems={ALIGN_ITEMS.CENTER}
       borderWidth={1}
-      borderRadius={SIZES.SM}
+      borderRadius={BORDER_RADIUS.SM}
       paddingLeft={leftAccessory ? 4 : 0}
       paddingRight={rightAccessory ? 4 : 0}
       onClick={handleClick}
```

Comment:
I could move these into the text-field-base updates into the house keeping pr for text-field-base if it's cleaner I'll work on that next.